src/accessory.ts

```diff
@@ -93,14 +96,32 @@ class PurpleAirSensor implements AccessoryPlugin {
   }
 
   update() {
-    const url = 'https://www.purpleair.com/json?show=' + this.sensor;
+    const url = 'https://www.purpleair.com/json';
+
+    const axiosInstance = axios.create();
+
+    axiosInstance.interceptors.request.use((request) => {
+      // write down your request intercept.
+      return AxiosLogger.requestLogger(request, {
+        logger: this.log
+      });
+    });
 
     if (this.lastReading !== undefined && this.lastReading.updateTimeMs > Date.now() - PurpleAirSensor.MIN_UPDATE_INTERVAL_MS) {
       this.log(`Skipping a fetch because the last update was ${Date.now() - this.lastReading.updateTimeMs} ms ago`);
     } else {
-      this.log(`Fetching ${url}`);
+      this.log(`Fetching`);
```

Comment:
just logging "Fetching" might not be very useful.

Reply:
Good call, removed